Documentation/commands.md

```diff
@@ -47,4 +47,7 @@ lookup the PATH first to see if there's a matching install, etc.
     "version": "8.0",
     "mode": "aspnetcore"
 }
-```
\ No newline at end of file
+```
+
+Note: If you are a developer for a highly used extension or an extension that is provided by Microsoft, you may be in our 'skip' list to enhance performance.
+Please check the list if this is not working for you and ask us to exclude you: [Skipped Extensions](https://github.com/dotnet/vscode-dotnet-runtime/blob/main/vscode-dotnet-runtime-library/src/Acquisition/JsonInstaller.ts).
```

Comment:
nit: Generally, markdown documents within the same repo should always use relative paths to those documents. That way, this link from any branch always points to the document version within that branch.
```suggestion
Please check the list if this is not working for you and ask us to exclude you: [Skipped Extensions](vscode-dotnet-runtime-library/src/Acquisition/JsonInstaller.ts).
```

Reply:
That's a great point. Thank you, fixed.